include/TPP/Dialect/Xsmm/XsmmUtils.h

```diff
@@ -11,7 +11,9 @@
 
 #include "TPP/Dialect/Xsmm/XsmmEnum.h"
 #include "TPP/Dialect/Xsmm/XsmmOps.h"
-#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include "TPP/IR/StructuredOpMatcher.h"
```

Comment:
Do not add more headers to headers if you're not using it in the actual header. Add it to the CPP file that uses it.